packages/cli/src/options/beaconNodeOptions/api.ts

```diff
@@ -1,7 +1,12 @@
-import {defaultOptions, IBeaconNodeOptions, allNamespaces} from "@lodestar/beacon-node";
-import {CliCommandOptions} from "../../util/index.js";
+import { defaultOptions, IBeaconNodeOptions, allNamespaces } from "@lodestar/beacon-node";
+import { CliCommandOptions } from "../../util/index.js";
 
-const enabledAll = "*";
+// Before: Original value was "*"
+// The original value "*" is not very friendly to use in bash scripts as it's interpreted as a wildcard character.
+// This change replaces "*" with "all" to improve compatibility with bash scripts.
+
+// After: Changed to "all" for better compatibility with bash scripts
+const enabledAll = "all";
```

Comment:
there is not need to comment the previous behavior
```suggestion
const enabledAll = "all";
```